scripts/update_uat.sh

```diff
@@ -12,10 +12,13 @@ cd ${PROJECT_ROOT}
 echo "Disabling config_readonly."
 touch disable-config-readonly
 
-./vendor/bin/drush cache:clear bin config --yes
-./vendor/bin/drush updatedb --yes
-./vendor/bin/drush cs-update --discard-overrides --yes
-./vendor/bin/drush cache-rebuild --yes
+./vendor/bin/drush cache:clear bin config --yes &&
```

Comment:
We don't need to concatenate this with the &&, as the -e parameter makes sure that it fails on the first error